Formula/odin.rb

```diff
@@ -18,9 +17,6 @@ class Odin < Formula
   end
 
   depends_on "llvm@14"
-  # Build failure on macOS 10.15 due to `__ulock_wait2` usage.
-  # Issue ref: https://github.com/odin-lang/Odin/issues/1773
-  depends_on macos: :big_sur
 
   fails_with gcc: "5" # LLVM is built with GCC
```

Comment:
I think I've seen other formulas drop this, but I'm not sure in which case it's dropped or not.

Reply:
So, should the `fails_with` be dropped or not?